styles/globals.scss

```diff
@@ -128,6 +128,27 @@
   font-style: normal;
   font-display: swap;
 }
+@font-face {
+  font-family: 'OpenSansCondensed';
+  src: url('../assets/fonts/OpenSansCondensed/OpenSansCondensed-Bold.ttf');
+  font-weight: 700;
+  font-style: normal;
+  font-display: swap;
+}
+@font-face {
+  font-family: 'OpenSansCondensed';
+  src: url('../assets/fonts/OpenSansCondensed/OpenSansCondensed-Light.ttf');
+  font-weight: 300;
+  font-style: normal;
+  font-display: swap;
+}
+@font-face {
+  font-family: 'OpenSansCondensed';
+  src: url('../assets/fonts/OpenSansCondensed/OpenSansCondensed-LightItalic.ttf');
+  font-weight: 300;
+  font-style: normal;
+  font-display: swap;
+}
```

Comment:
both fonts declarations have the same attributes but they're actually different, maybe the last one should be ` font-style: italic`

Reply:
you are right, done